3.7/init.sh

```diff
@@ -4,27 +4,27 @@ OSMFILE=${PROJECT_DIR}/data.osm.pbf
 
 if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
   echo "Downloading Wikipedia importance dump"
-  curl https://www.nominatim.org/data/wikimedia-importance.sql.gz -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
+  curl https://www.nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
 else
   echo "Skipping optional Wikipedia importance import"
 fi;
 
 if [ "$IMPORT_GB_POSTCODES" = "true" ]; then
-  curl https://www.nominatim.org/data/gb_postcode_data.sql.gz -o ${PROJECT_DIR}/gb_postcode_data.sql.gz
+  curl https://www.nominatim.org/data/gb_postcode_data.sql.gz -L -o ${PROJECT_DIR}/gb_postcode_data.sql.gz
 else \
   echo "Skipping optional GB postcode import"
 fi;
 
 if [ "$IMPORT_US_POSTCODES" = "true" ]; then
-  curl https://www.nominatim.org/data/us_postcode_data.sql.gz -o ${PROJECT_DIR}/us_postcode_data.sql.gz
+  curl https://www.nominatim.org/data/us_postcode_data.sql.gz -L -o ${PROJECT_DIR}/us_postcode_data.sql.gz
 else
   echo "Skipping optional US postcode import"
 fi;
 
 
 if [ "$PBF_URL" != "" ]; then
 	echo Downloading OSM extract from "$PBF_URL"
-	curl -L "$PBF_URL" --create-dirs -o $OSMFILE
+	curl -L "$PBF_URL" -L --create-dirs -o $OSMFILE
```

Comment:
This one already has the option at the beginning. Not sure if it will cause an error when set twice.

Reply:
thanks for noticing